### ISSUE TITLE
Fix donate link in readme

### ISSUE THREAD

**Issue by hubdotcom** [birdsarah/wp-recent-revisions#14]
http://www.sarahbird.org/recent-revisions cannot see where to donate

You can also make a direct link from paypal here: https://www.paypal.com/webapps/mpp/get-started/donate-button

**Comment by birdsarah** [comment]
I guess I took the link down.

I've had one person donate, which was super awesome, but I'm not that keen on putting it back up.

I feel like keeping this plugin alive is a bare minimum contribution back to the fact that wordpress and the rest of the community is awesome!

Lets delete that link.

**Comment by hubdotcom** [comment]
Yeah, that's the way it goes - maybe it grows and snowballs and you can launch a premium version but that's not the primary aim. Bottom line - do it for love not money!

**Comment by birdsarah** [comment]
@hubdotcom, i've just realized it's been a year since we started all these conversations. I clearly don't have the bandwidth to keep them going. Would you like to take over the codebase?
